lib/pbench/test/unit/server/test_user_management_cli.py

```diff
@@ -0,0 +1,251 @@
+import datetime
+
+import pytest
+import responses
+from click.testing import CliRunner
+
+from pbench.cli.server.user_management import (
+    USER_LIST_HEADER_ROW,
+    USER_LIST_ROW_FORMAT,
+    user_create,
+    user_delete,
+    user_list,
+    user_update,
+)
+
+
+class TestUserManagement:
+    USER_SWITCH = "--username"
+    PSWD_SWITCH = "--password"
+    PSWD_PROMPT = "Password: "
+    EMAIL_SWITCH = "--email"
+    FIRST_NAME_SWITCH = "--first-name"
+    LAST_NAME_SWITCH = "--last-name"
+    ROLE_SWITCH = "--role"
+    USER_TEXT = "test_user"
+    PSWD_TEXT = "password"
+    EMAIL_TEXT = "test@domain.com"
+    FIRST_NAME_TEXT = "First"
+    LAST_NAME_TEXT = "Last"
+    URL = "http://pbench.example.com/api/v1"
+    ENDPOINT = "/register"
+
+    @staticmethod
+    @responses.activate
```

Comment:
I don't see any `responses` mocks being set up, so what's `@responses.activate` accomplishing here? Or am I missing something implicit somewhere?

Reply:
Without any mock-requests, I believe that `@responses.activate` will catch and report any requests which are made (i.e., ones which should have had mocks or are otherwise unexpected and/or erroneous).

But, given that the code being tested here is talking to the database directly and not making any HTTP requests, there _would_ be an argument for not having the `requests` package present and operating.  (Of course, the reverse argument works too:  this code shouldn't be making any HTTP requests, and we'll ensure that using `@responses.activate`!  :-D )

Reply:
Huh.